cli/src/commands/sidekick.rs

```diff
@@ -446,11 +448,23 @@
 					"error": "No Rivet token found, please do the sign in process",
 				})),
 			)?;
+			return Err(Error::msg(
+				"No Rivet token found, please do the sign in process",
+			));
 		}
 
 		Ok(())
 	}
 
+	pub fn check_login_state(&self, token: &Option<String>) -> Result<()> {
```

Comment:
I would recommend validating the token like [here](https://github.com/rivet-gg/cli/blob/fd359593cd60b736f6d34d25f33c4b2fe925f7bd/core/src/ctx.rs#L44), unless I'm misunderstanding the use case.